tests/pkunit_test.py

```diff
@@ -154,8 +154,10 @@ def test_pkre_convert():
         pkre(r, s)
 
 def test_xlsx_to_csv_conversion():
+    from pykern import pkio
     from pykern.pkunit import file_eq, data_dir
 
+    pkio.write_binary(data_dir().join('example.xlsx'), pkio.read_binary(data_dir().join('actual_excel.xlsx')))
```

Comment:
Rename actual_excel.xlsx to example.xlsx. Don't write into data_dir during a test. Always write into work_dir. Then use `pkio.write_binary(empty_work_dir().join('example.xlsx'), data_dir().join('example.xslx'))`